Moved onSuccess call after render call

Flave opened the pull request:
In order to get the dimensions (e.g. clientHeight, clientWidth) of the embed it needs to be rendered to the screen. This is why the onSuccess call is better made after the internal state update that causes the rerender.
Alternatively there could be an additional callback such as `onRender`. But since I didn’t see any particular advantage of placing the onSuccess call before the render, it is easier to just move it to the end. Hope this makes sense :)

sugarshin commented:
Thank you for this contribution!
Sounds good, but it's breaking change. that's way `onRender` option is good your idea. (I think `onAfterRender` or `onRenderd` is more better)
Can you fix it?

e.x

```js
this.props.onSuccess && this.props.onSuccess(response)
this.setState(
  { __html: response.html },
  () => {
    window.instgrm.Embeds.process()
    this.props.onAfterRender && this.props.onAfterRender()
  }
)
```

Flave commented:
Yes sounds good. I added an onAfterRender prop. I added the prop to the `omitComponentProps` function without knowing exactly what it does. I'm not really familiar with TypeScript (that's what this is, right?) so I hope I didn't break anything :)

sugarshin commented:
Thanks! I will few fix it later.

sugarshin commented:
@Flave Merged. thanks
